def a():
    try:
        var = "a" + 5
    except Exception as exc:
        return exc
